### PR TITLE
Remove hard UNIX dependency

### PR DESCRIPTION
**Pull request by atward**
Optimistically set non-blocking I/O if fcntl is available.

This code reminds me of something. ¬_¬

### CODE REVIEW

**Comment by atward** [comment]
Can I get a version bump + pypi if/when this goes out?

File: delfick_app.py [hunk 1]
```diff
@@ -602,6 +601,15 @@ def read_non_blocking(stream):
             else:
                 break
 
+def set_non_blocking_io(fh):
+    if sys.platform in ('win32'):
```
Comment (delfick) [inline delfick_app.py]: shouldn't this be ('win32', ) ?

(missing comma)

Comment (atward) [reply delfick_app.py]: Yes. Made it a list instead.

**Review by delfick — APPROVED** [review]

**Comment by delfick** [comment]
0.9.5 has been released

**Pull request merged**